bodhi/tests/server/base.py

```diff
@@ -52,15 +48,13 @@
 class BaseTestCase(unittest.TestCase):
     def setUp(self):
         bugs.set_bugtracker()
-        engine = create_engine(DB_PATH)
-        # We want a special session that lasts longer than a transaction
-        self.Session = scoped_session(
-            sessionmaker(bind=engine, extension=ZopeTransactionExtension(keep_session=True)))
-        log.debug('Creating all models for %s' % engine)
-        models.Base.metadata.bind = engine
-        models.Base.metadata.create_all(engine)
+        self.config = {'sqlalchemy.url': DB_PATH}
+        self.engine = initialize_db(self.config)
+        self.Session = Session
```

Comment:
I just didn't want to get into the business of adjusting tests more than I absolutely had to. The tests really need a clean-up, but I think we want to bite that off separately.